### ISSUE TITLE
prep for release 1.3

### ISSUE THREAD

**Issue by mhw-at-yg** [JamesRamm/archook#18]
Hello, new maintainer Matt Wilkie here.  I'm planning to issue a new release, chiefly to take advantage of the support for ArcGIS Pro from #16 courtesy of @logan-pugh. Please raise anything you think should be addressed or included, either as comment here or new issue.

- [x] #1 cannot find _base module err
- [x] #19 pep8 formatting
- [x] #20, #21 numpy import err

_I treat 1st comment as wiki and update periodically to reflect issue status_

**Comment by logan-pugh** [comment]
I think it would be good to standardize on a code style/linter; I like flake8 (with a few rules turned off), and 4 spaces instead of 2.

I'll submit another PR with my suggestions. edit: #19

**Comment by logan-pugh** [comment]
See also #20 and #21 -- I *think* that's it from me 😉

**Comment by mhw-at-yg** [comment]
All of the things identified for 1.3 release have been done. @logan-pugh  you've done most of the new feature work, do you see anything else that should be done before making a release?

**Comment by mhw-at-yg** [comment]
I pushed a release candidate 1.3.0rc tag.

**Comment by jesegal** [comment]
I tried this version with pyinstaller and got the failure below during runtime.
Am I jumping the gun, so to speak?

  File "site-packages\archook\__init__.py", line 14, in <module>
  File "site-packages\archook\__init__.py", line 12, in get_version
  File "site-packages\pkg_resources\__init__.py", line 482, in get_distribution
  File "site-packages\pkg_resources\__init__.py", line 358, in get_provider
  File "site-packages\pkg_resources\__init__.py", line 901, in require
  File "site-packages\pkg_resources\__init__.py", line 787, in resolve
pkg_resources.DistributionNotFound: The 'archook' distribution was not found and is required by the application

**Comment by mhw-at-yg** [comment]
@jesegal nope, not jumping the gun, please open as a a new issue though.

**Comment by cooperjaXC** [comment]
@mhw-at-yg Any updates on this? I dutifully followed advice from #31 embedding this neat tool into my own packages and products, and it is working great. As of now though, the [PyPi version](https://pypi.org/project/archook/) is still `v1.2` from 2017. Even if the current `v1.3.dev0` is pushed, there are valuable ArcGIS Pro enhancements that would be great to have available through the usual `pip install` method. It may fix other issues like #30 too. Appreciate the help.

**Comment by mhw-at-yg** [comment]
thanks for the nudge @cooperjaXC . I've added this to my to-look-at for tomorrow

**Comment by maphew** [comment]
@cooperjaXC if you pip install from github does that still work for you? I'm not making much headway against #34, but maybe that only affects conda users like myself and isn't a general issue.

     pip install https://github.com/JamesRamm/archook/archive/refs/heads/master.zip

**Comment by cooperjaXC** [comment]
> @cooperjaXC if you pip install from github does that still work for you? I'm not making much headway against #34, but maybe that only affects conda users like myself and isn't a general issue.
> 
> ```
>  pip install https://github.com/JamesRamm/archook/archive/refs/heads/master.zip
> ```

@maphew It works fine for me, but I don't `conda install`, I only `pip install` using the python interpreter that comes with anaconda. I use the command suggested in https://github.com/JamesRamm/archook/issues/31#issuecomment-736935126, and it works in both command line and `requirements.txt` installations. No `numpy` errors.

**Comment by mhw-at-yg** [comment]
Closing as too old and likely either solved or too many changes to address cleanly from this issue. Please open new issues for any parts that are still relevant.
